providers/implementations/keymgmt/dh_kmgmt.c

```diff
@@ -196,5 +214,6 @@ const OSSL_DISPATCH dh_keymgmt_functions[] = {
     { OSSL_FUNC_KEYMGMT_IMPORTKEY, (void (*)(void))dh_importkey },
     { OSSL_FUNC_KEYMGMT_EXPORTKEY, (void (*)(void))dh_exportkey },
     { OSSL_FUNC_KEYMGMT_FREEKEY, (void (*)(void))DH_free },
+    { OSSL_FUNC_KEYMGMT_GET_KEY_PARAMS,  (void (*) (void))dh_get_key_params },
```

Comment:
Is this supposed to have a get_domain_params entry?

Reply:
Yes.  The same goes for the DSA implementation, of course

Reply:
Fixed